go/test/endtoend/README.md

```diff
@@ -31,22 +32,13 @@ In general the cluster is build in following order
 
 A good example to refer will be  go/test/endtoend/clustertest
 
-## Progress
-So far we have converted the following Python end to end test cases
-- Keyspace tests
-- mysqlctl tests
-- sharded tests
-- tabletmanager tests
-- vtgate v3 tests
-- Inital sharding
-- resharding
-- vsplit
-- reparent
- 
-### In-progress 
-- Backup
-- Encryption
-
-After a Python test is migrated in Go it will be removed from end to end ci test run by updating the shard value to 5 in `test/config.json`
+### Pre-Requisite 
+Make sure you have vitess binary available in bin folder. If not, then you can run `./bootstrap.sh` follow by `make build` & `source build.env`.
 
-    
+To make it easier to re-run test please add following to you bash profile. 
+ 
+```
+export VTROOT=/<vitess path>/vitess
+export VTDATAROOT=${VTROOT}/vtdataroot
+export PATH=${VTROOT}/bin:${PATH}
```

Comment:
Is this strictly required? build.env should be able to guess these. In future we may even be able to deprecate VTROOT as it can be implied.

Reply:
For now, it needs vitess binaries to be in PATH and we should also have VTDATAROOT . So when `VTROOT` deprecates, we can change this .